packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -360,21 +354,27 @@ const SavedChartsHeader: FC = () => {
                 />
             )}
             {isDeleteDialogOpen && savedChart?.uuid && (
-                <DeleteActionModal
-                    isOpen={isDeleteDialogOpen}
-                    onClose={() => setIsDeleteDialogOpen(false)}
+                <ChartDeleteModal
                     uuid={savedChart.uuid}
-                    name={savedChart.name}
-                    isChart
-                    isExplorer
-                />
-            )}
-            {isMoveToSpaceModalOpen && savedChart?.uuid && (
-                <MoveToSpaceModal
                     isOpen={isDeleteDialogOpen}
-                    onClose={() => setIsMoveToSpaceModalOpen(false)}
-                    uuid={savedChart.uuid}
-                    isChart
+                    onClose={() => setIsDeleteDialogOpen(false)}
+                    onConfirm={() => {
+                        // FIXME: THIS IS A COPY OF THE CODE
+                        // NOT SURE WHAT IT DOES...
+                        history.listen((location, action) => {
+                            if (action === 'POP') {
+                                if (location.pathname.includes('/tables/')) {
+                                    history.push(
+                                        `/projects/${projectUuid}/tables`,
+                                    );
+                                }
+                            }
+                        });
```

Comment:
this is just a copy paste but not sure what it does. can someone comment on this?

Reply:
The code was added here https://github.com/lightdash/lightdash/commit/630431e9f386425b11d5cb3d9494809469c1a9a7#diff-ac5989858682adb7f5ab937acb87d1298c30b95a7ac18b1817d00abb6df3a54eR105 . It seems it is doing some redirection when you remove charts...

Reply:
gonna leave it as is 🙌  was just curious